### PR TITLE
tech talk 2015-02-12

### PR DESCRIPTION
**Pull request by frankhale3**

### CODE REVIEW

**Comment by dzmitry-tsurankou** [comment]
was there anyone who looked at it? cc/ @Daniel-Awai

File: Demo/posroot.cpp [hunk 1]
```diff
@@ -0,0 +1,25 @@
+// compute  positive root of a quadratic equation
+
+#include <iostream>
+#include <iomanip>
+#include <math.h>
+using namespace::std;
+
+int main () 
+{
+   float a, b, c, r;
```
Comment (dzmitry-tsurankou) [inline Demo/posroot.cpp]: indentation is 3 spaces here

Comment (frankhale3) [inline Demo/posroot.cpp]: dzmitry, this is nothing more than a quick-and-dirty c++ translation of the "first fortran program" for the sake of those who cannot read the original fortran.  it is not intended to be part of a software development project, but rather an aid in underscoring a fortran artifact from the history of computing.

Comment (frankhale3) [inline Demo/posroot.cpp]: indentation is irrelevant.  this is nothing more than a quick-and-dirty c++ translation of the first program written in fortran.  it is an aid to understanding an important artifact in the history of computing, for those who cannot read fortran.  again, indentation is irrelevant.

**Comment by frankhale3** [comment]
dzmitry, these are demo and sample codes that go along with the talk on FP given on february 12.  they are here for archival and reference purposes.

**Comment by frankhale3** [comment]
dzmitry, if you are in any way concerned with my uses of this github repo to archive my tech talks for the club, it is easy enough for me to stop doing that.  I only archive materials here so that they can be easily accessed by club members who might want to refer to the presentations (keynote and pdf formats) and related demo codes.  as far as i know, i am a team of one in this work.  however, i can backup my materials elsewhere, and keep them on my own systems, if that makes management of github easier for you.

**Comment by Daniel-Awai** [comment]
I'm wondering why I'm only getting updates on this now. I really like github to be a place that we can store and share resources, but I would like them to adheard to the cc's that we have for archival purposes. *start shamless plug for vim* gg=G will auto indent to the specifications that you have in your vimrc or the default spec'd by vim.

**Pull request merged**